docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -231,6 +231,48 @@ docker run \
 
 The default Intel Quote Provider config is available in [Intel SGX Github repository](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/qcnl/linux/sgx_default_qcnl.conf).
 
+## Migrate from EPID Attestation to DCAP Attestation
+EPID attestation support has been discontinued in newer processors (Intel Xeon
+3rd generation onwards). All nodes using EPID attestation should migrate to DCAP
+Attestation.
+
+:::info
+
+Validators, please hold for further instructions. We will coordinate the
+migration in phases to ensure that during the migration the network remains
+operational.
+
+:::
+
+:::info
+
+In case you are running AESM service on Docker please see [these instructions].
+
+:::
+
+To migrate to the DCAP Attestation run the following commands:
```

Comment:
Before performing these steps one should check if their hardware supports DCAP as per the docs above.